Mathlib/CategoryTheory/Shift/CommShiftTwo.lean

```diff
@@ -90,6 +91,10 @@ class CommShift₂ {M : Type*} [AddCommMonoid M] [HasShift C₁ M] [HasShift C
           (((G.obj X₁).commShiftIso n).hom.app X₂)⟦m⟧' ≫
             (shiftComm ((G.obj X₁).obj X₂) m n).inv ≫ (h.ε m n).val.app _
 
+alias commShift₂_comm := CommShift₂.comm
```

Comment:
I take that this alias is here so that dot notation works? If that is the case, can you please mention that in
a comment above the alias declaration?

Reply:
Thanks for the suggestion.